app/javascript/app/components/indicator/indicator-component.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { themr } from 'react-css-themr';
+import cx from 'classnames';
+import Input from 'components/input';
+import Dropdown from 'components/dropdown';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import deleteIcon from 'assets/icons/info.svg';
+
+import squareButtonTheme from 'styles/themes/button/button-square.scss';
+import blueIconTheme from 'styles/themes/icon/icon-blue.scss';
+import styles from './indicator-styles.scss';
+
+const Indicator = ({
+  title,
+  fieldType,
+  theme,
+  inputUnit,
+  label,
+  dropdownOptions
+}) => (
+  <div
+    className={cx(theme.wrapper, {
+      [theme.textArea]: fieldType === 'textarea'
+    })}
+  >
+    <p className={theme.indicatorTitle}>{title}</p>
+    {fieldType === 'dropdown' ? (
+      dropdownOptions && (
+        <Dropdown options={dropdownOptions} unit={inputUnit} label={label} />
+      )
+    ) : (
+      <Input inputType={fieldType} unit={inputUnit} label={label} />
```

Comment:
Don't we have an example with two inputs? Maybe we should just render children instead of this ternary

![image](https://user-images.githubusercontent.com/9701591/38941702-6e3f26d6-432d-11e8-8017-f86fc01e0cd0.png)